lib/nacelle-pushowl-plugin.js

```diff
@@ -107,14 +145,24 @@ export default function (ctx) {
       return
     }
 
+    const subscriberToken = localStorage.getItem('pushowl_subscriber_token')
+    document.cookie = `pushowl_subscriber_token=${subscriberToken}`
```

Comment:
why set the cookie here?
1. It needs to be set when we get subs token. Here it will get unnecessary called on every store mutation
2. this cookie doesnt look like its set on `.domain.com` to be able to be read on `subdomain.domain.com`

I think a better place to do this would be SDK since we also don't tie ourself to Nacelle

Reply:
Let me move this to the sdk. Also I don't think '.' before `domain.com` is needed (see source of https://dupexpansion.surge.sh/)

Reply:
Removed from here.